src/js/configuration_panel/configuration-panel.js

```diff
@@ -89,7 +89,8 @@ https://github.com/fluid-lab/gamepad-navigator/blob/master/LICENSE
                 zoomIn: "Zoom-in on the active web page",
                 zoomOut: "Zoom-out on the active web page",
                 maximizeWindow: "Maximize the current browser window",
-                restoreWindowSize: "Restore the size of current browser window"
+                restoreWindowSize: "Restore the size of current browser window",
+                reopenTabOrWindow: "Re-open the closed tab or window"
```

Comment:
I know that Chrome itself uses similar wording, but I'd prefer to add "last" here, as in "last closed".